{{cookiecutter.project_slug}}/frontend/package.json

```diff
@@ -24,6 +24,8 @@
     ]
   },
   "dependencies": {
+    "@sentry/browser": "^7.73.0",
```

Comment:
@nvernooy Maybe we should use [@sentry/react](https://www.npmjs.com/package/@sentry/react) instead?

> This package is a wrapper around @sentry/browser, with added functionality related to React.